languages/csharp/exercises/concept/casting/.docs/introduction.md

```diff
@@ -0,0 +1,37 @@
+Casting and type conversion are different ways of changing an expression from one data type to another.
+
+An expression can be cast to another type with the cast operator `(<tupe>)`.
+
+```csharp
+long l = 1000L;
+
+int i = (int)l;
+
+object o = new Random();
+
+Random r = (Random)o;
```

Comment:
Maybe group them to show they belong together visually.

```suggestion
long l = 1000L;
int i = (int)l;

object o = new Random();
Random r = (Random)o;
```

Reply:
Done in new PR

Reply:
There is a new PR?